scripts/take_bias.sh

```diff
@@ -30,15 +30,15 @@ P=$1
 SAVE_FILE=${2:-"bias-%Y%m%d-%H%M%S.cr2"}
 echo "Taking bias frames on port ${P}: ${SAVE_FILE}"
 
-NUM_FRAMES=${4:-1}
-SHUTTER_INDEX=${3:-52} # 1/4000s on EOS 100D
+NUM_FRAMES=${3:-1}
+SHUTTER_INDEX=${4:-52} # 1/4000s on EOS 100D
```

Comment:
How about a TODO or Issue that we should allow exposure time to be specified, or how to get the list of indices and corresponding exposure times supported by the camera?

Reply:
I think this is what Aru said she modified so I was waiting to see if we get that.